webhook_server_container/libs/github_api.py

```diff
@@ -2079,6 +2079,14 @@ def owners_data_for_changed_files(self) -> dict[str, dict[str, Any]]:
 
         return data
 
+    def _get_owner_dir_name_for_changed_folder(self, changed_folder: str) -> str:
```

Comment:
please get here Path, not str and return Path as well

Reply:
I have changed it totally. Now I am returning the combined approvers, reviewers dict. Do you still want me to pass the Path to the function? Let me know I will update accordingly.